atomic_reactor/plugins/build_source_container.py

```diff
@@ -81,9 +94,12 @@ def run(self):
             cmd.append('{}'.format(source_data_dir))
 
         if remote_source_exists:
+            self.split_remote_sources_to_subdirs(remote_source_data_dir)
             drivers.add('sourcedriver_extra_src_dir')
-            cmd.append('-e')
-            cmd.append('{}'.format(remote_source_data_dir))
+
+            for source_subdir in sorted(os.listdir(remote_source_data_dir)):
```

Comment:
Could `split_remote_sources_to_subdirs()` return list of abspaths instead of doing os.listdir twice?

Reply:
sure